src/components/errorNotification.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  onNotificationClose: () => void,
+  isErrorOccuring: boolean,
+};
+export const ErrorNotification: React.FC<Props> = ({
+  onNotificationClose,
+  isErrorOccuring,
+}) => {
+  const onCloseHandler = () => {
+    onNotificationClose();
+  };
+
+  return (
+    <div className={classnames('notification', 'is-danger',
+      'is-light', 'has-text-weight-normal', { hidden: isErrorOccuring })}
+    >
+      <button
+        type="button"
+        className="delete"
+        onClick={onCloseHandler}
```

Comment:
you can use the `onNotificationClose` directly here without additional wrapper.
```suggestion
        onClick={onNotificationClose}
```